src/main/java/com/sap/sgs/phosphor/fosstars/nvd/data/CVE.java

```diff
@@ -46,97 +55,149 @@
-
-  @JsonProperty("data_type")
-  public String getDataType() {
-    return dataType;
-  }
-
-  @JsonProperty("data_type")
-  public void setDataType(String dataType) {
-    this.dataType = dataType;
-  }
-
   @JsonProperty("data_format")
-  public String getDataFormat() {
+  public CVE.DataFormat getDataFormat() {
     return dataFormat;
   }
 
-  @JsonProperty("data_format")
-  public void setDataFormat(String dataFormat) {
-    this.dataFormat = dataFormat;
-  }
-
-  @JsonProperty("data_version")
-  public String getDataVersion() {
-    return dataVersion;
-  }
-
-  @JsonProperty("data_version")
-  public void setDataVersion(String dataVersion) {
-    this.dataVersion = dataVersion;
-  }
-
   @JsonProperty("CVE_data_meta")
-  public CVEDataMeta getCveDataMeta() {
-    return cveDataMeta;
-  }
-
-  @JsonProperty("CVE_data_meta")
-  public void setCveDataMeta(CVEDataMeta cveDataMeta) {
-    this.cveDataMeta = cveDataMeta;
+  public CveMetaData getCveDataMeta() {
+    return cveDataMetaData;
   }
 
   @JsonProperty("affects")
   public Affects getAffects() {
     return affects;
   }
 
-  @JsonProperty("affects")
-  public void setAffects(Affects affects) {
-    this.affects = affects;
-  }
-
-  @JsonProperty("problemtype")
-  public ProblemType getProblemType() {
-    return problemType;
-  }
-
-  @JsonProperty("problemtype")
-  public void setProblemType(ProblemType problemType) {
-    this.problemType = problemType;
-  }
-
   @JsonProperty("references")
   public References getReferences() {
     return references;
   }
 
-  @JsonProperty("references")
-  public void setReferences(References references) {
-    this.references = references;
-  }
-
   @JsonProperty("description")
   public Description getDescription() {
     return description;
   }
 
-  @JsonProperty("description")
-  public void setDescription(Description description) {
-    this.description = description;
+  public enum DataFormat {
+
+    MITRE("MITRE");
+
+    private static final Map<String, CVE.DataFormat> CONSTANTS = new HashMap<>();
+
+    static {
+      for (CVE.DataFormat c : values()) {
+        CONSTANTS.put(c.value, c);
+      }
+    }
+
+    private final String value;
+
+    DataFormat(String value) {
+      this.value = value;
+    }
+
+    @JsonCreator
+    static CVE.DataFormat fromValue(String value) {
+      CVE.DataFormat constant = CONSTANTS.get(value);
+      if (constant == null) {
+        throw new IllegalArgumentException(value);
+      } else {
+        return constant;
+      }
+    }
+
+    @Override
+    public String toString() {
+      return this.value;
+    }
+
+    @JsonValue
+    public String value() {
+      return this.value;
+    }
+
   }
 
-  @JsonAnyGetter
-  public Map<String, Object> getAdditionalProperties() {
-    return this.additionalProperties;
+  public enum DataType {
+
+    CVE("CVE");
+
+    private static final Map<String, com.sap.sgs.phosphor.fosstars.nvd.data.CVE.DataType> CONSTANTS
```

Comment:
Can this package be in the import section? Or is it better to change the name of the enum?

Reply:
Yeah, I'll make it shorter.

Reply:
Done.